src/plays/image-carousel/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Image Carousel
+
+Image Carousel that has click navigation for desktop and touch(swipe) navigation for mobile
+
+## Play Demographic
+
+- Language: ts
+- Level: Beginner
+
+## Creator Information
+
+- User: harshsinghatz
+- Gihub Link: https://github.com/harshsinghatz
+- Blog: https://harshsinghatz.hashnode.dev/
+- Video: 
+
+## Implementation Details
+
+Update your implementation idea and details here
```

Comment:
Could you please update some implementation details here? We want any beginners to learn the technical part as well from the read me.

Reply:
@atapas Done!!